application/src/main/java/org/togetherjava/tjbot/features/moderation/TransferQuestionCommand.java

```diff
@@ -69,61 +69,64 @@ public void onMessageContext(MessageContextInteractionEvent event) {
         String authorId = event.getTarget().getAuthor().getId();
+        String mostCommonTag = defaultTags.get(0);
 
-        TextInput transferQuestionTitle =
-                TextInput.create(TRANSFER_QUESTION_TITLE_ID, "Title", TextInputStyle.SHORT)
-                    .setMaxLength(70)
-                    .setMinLength(8)
-                    .setValue(createTitle(originalMessage))
-                    .build();
-
-        TextInput transferQuestionInput = TextInput
-            .create(TRANSFER_QUESTION_INPUT_ID, "Transfer question menu", TextInputStyle.PARAGRAPH)
-            .setValue(originalMessage)
-            .setRequiredRange(3, 2000)
+        TextInput modalTitle = TextInput.create(MODAL_TITLE_ID, "Title", TextInputStyle.SHORT)
+            .setMaxLength(70)
+            .setMinLength(4)
+            .setPlaceholder("describe question in short")
+            .setValue(createTitle(originalMessage))
             .build();
 
-        TextInput transferQuestionTag = TextInput
-            .create(TRANSFER_QUESTION_TAG, "Transfer question tags", TextInputStyle.SHORT)
-            .setValue("Java")
+        TextInput modalInput =
+                TextInput.create(MODAL_INPUT_ID, "Question", TextInputStyle.PARAGRAPH)
+                    .setValue(originalMessage)
+                    .setRequiredRange(3, 2000)
+                    .setPlaceholder("contents of question")
+                    .build();
+
+        TextInput modalTag = TextInput.create(MODAL_TAG, "Most fitting tag", TextInputStyle.SHORT)
+            .setValue(mostCommonTag)
+            .setPlaceholder("suitable tag for question")
             .build();
 
-        String transferQuestionModalComponentID =
+        String modalComponentID =
                 generateComponentId(authorId, originalMessageId, originalChannelId);
-        Modal transferModal =
-                Modal.create(transferQuestionModalComponentID, "transfer question menu")
-                    .addActionRow(transferQuestionTitle)
-                    .addActionRow(transferQuestionInput)
-                    .addActionRow(transferQuestionTag)
-                    .build();
+        Modal transferModal = Modal.create(modalComponentID, "Transfer this question")
+            .addActionRow(modalTitle)
+            .addActionRow(modalInput)
+            .addActionRow(modalTag)
+            .build();
 
         event.replyModal(transferModal)
             .queue(success -> logger.debug(
                     "{} with id: {}  triggered the transfer action on forumPost with id: {}",
-                    event.getUser().getName(), event.getUser().getId(), originalMessageId),
-                    failed -> {
-                    });
+                    event.getUser().getName(), event.getUser().getId(), originalMessageId));
     }
 
     @Override
     public void onModalSubmitted(ModalInteractionEvent event, List<String> args) {
         event.deferEdit().queue();
 
+        String authorId = args.get(0);
```

Comment:
author of what? the original post or the user who invokes the command? please clarify and adjust the variable accordingly

Reply:
perhaps `originalAuthorId`? to give the connection to the original message